easybuild/easyconfigs/p/Perl/Perl-5.28.0-GCCcore-7.3.0.eb

```diff
@@ -22,7 +22,7 @@ dependencies = [
 osdependencies = ['openssl', ('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
 
 # !! order of extensions is important !!
-# extensions updated on June 26th 2018
+# extensions updated on May 6th 2018
```

Comment:
Also, changing the date here isn't really correct, since you're just adding an extension, not updating the existing ones.

Just reverting to what it was makes more sense to me...